Javadoc errors in build

Currently (5f31669c) following [is reported](https://travis-ci.com/github/eclipse-ee4j/openmq/builds/187162661#L4253-L4260):
```
  [javadoc] Constructing Javadoc information...
  [javadoc] /home/travis/build/eclipse-ee4j/openmq/mq/main/mq-broker/broker-core/src/main/java/com/sun/messaging/jmq/jmsserver/persist/api/Store.java:55: error: package jakarta.inject does not exist
  [javadoc] import jakarta.inject.Singleton;
  [javadoc]                      ^
  [javadoc] /home/travis/build/eclipse-ee4j/openmq/mq/main/mq-broker/broker-core/src/main/java/com/sun/messaging/jmq/jmsserver/persist/api/Store.java:63: error: cannot find symbol
  [javadoc] @Singleton
  [javadoc]  ^
  [javadoc]   symbol: class Singleton
```

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/ConnectionConfiguration.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/ConnectionConfiguration.java
@@ -777,7 +777,7 @@ public class ConnectionConfiguration {
      * control properties.
      *
      * @see com.sun.messaging.ConnectionConfiguration#imqConsumerFlowLimit
-     * @see com.sun.messaging.ConnectionConfiguration#imqConsumerFlowThreshold"
+     * @see com.sun.messaging.ConnectionConfiguration#imqConsumerFlowThreshold
      */
     public static final String imqConsumerFlowLimitPrefetch = "imqConsumerFlowLimitPrefetch";
 

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/Connection.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/Connection.java
@@ -80,7 +80,6 @@ public interface Connection extends jakarta.jms.Connection {
      * Remove a MQ consumer event listener from the current connection.
      *
      * @param dest the destination on which addConsumerEventListener() was called previously
-     * @param listener EventListener
      * @throws JMSException
      * @since 4.5
      */

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQIllegalStateRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQIllegalStateRuntimeException.java
@@ -75,7 +75,7 @@ public class MQIllegalStateRuntimeException extends jakarta.jms.IllegalStateRunt
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQInvalidClientIDRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQInvalidClientIDRuntimeException.java
@@ -74,7 +74,7 @@ public class MQInvalidClientIDRuntimeException extends jakarta.jms.InvalidClient
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQInvalidDestinationRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQInvalidDestinationRuntimeException.java
@@ -75,7 +75,7 @@ public class MQInvalidDestinationRuntimeException extends jakarta.jms.InvalidDes
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQInvalidSelectorRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQInvalidSelectorRuntimeException.java
@@ -74,7 +74,7 @@ public class MQInvalidSelectorRuntimeException extends jakarta.jms.InvalidSelect
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQMessageFormatRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQMessageFormatRuntimeException.java
@@ -74,7 +74,7 @@ public class MQMessageFormatRuntimeException extends jakarta.jms.MessageFormatRu
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQMessageNotWriteableRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQMessageNotWriteableRuntimeException.java
@@ -77,7 +77,7 @@ public class MQMessageNotWriteableRuntimeException extends MessageNotWriteableRu
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQResourceAllocationRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQResourceAllocationRuntimeException.java
@@ -73,7 +73,7 @@ public class MQResourceAllocationRuntimeException extends jakarta.jms.MessageFor
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQRuntimeException.java
@@ -74,7 +74,7 @@ public class MQRuntimeException extends jakarta.jms.JMSRuntimeException implemen
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQSecurityRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQSecurityRuntimeException.java
@@ -74,7 +74,7 @@ public class MQSecurityRuntimeException extends jakarta.jms.JMSSecurityRuntimeEx
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQTransactionInProgressRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQTransactionInProgressRuntimeException.java
@@ -75,7 +75,7 @@ public class MQTransactionInProgressRuntimeException extends jakarta.jms.Transac
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQTransactionRolledBackRuntimeException.java
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/MQTransactionRolledBackRuntimeException.java
@@ -75,7 +75,7 @@ public class MQTransactionRolledBackRuntimeException extends jakarta.jms.Transac
     /**
      * Specify whether this object is logged.
      *
-     * @param whether this object is logged
+     * @param state whether this object is logged
      */
     @Override
     public void setLogState(boolean state) {

--- a/mq/main/mq-client/src/main/java/com/sun/messaging/jms/package.html
+++ b/mq/main/mq-client/src/main/java/com/sun/messaging/jms/package.html
@@ -52,7 +52,7 @@ enable MQ JMS clients to perform finer
 grained error handling than that available with standard
 JMS exceptions.
 
-@see jakarta.jms jakarta.jms
+@see "Jakarta Messaging API"
 
 </BODY>
 </HTML>

--- a/mq/src/buildant/rules.xml
+++ b/mq/src/buildant/rules.xml
@@ -252,6 +252,7 @@
     <pathelement location="${packager.artifacts}/jakarta.transaction-api.jar"/>
     <pathelement location="${packager.artifacts}/jakarta.xml.soap-api.jar"/>
     <pathelement location="${packager.artifacts}/jakarta.activation-api.jar"/>
+    <pathelement location="${packager.artifacts}/jakarta.inject-api.jar"/>
     <pathelement location="${packager.artifacts}/jhall.jar"/>
     <pathelement location="${packager.artifacts}/hk2-api.jar"/>
     <pathelement location="${packager.artifacts}/hk2-runlevel.jar"/>

--- a/mq/src/buildant/rules.xml
+++ b/mq/src/buildant/rules.xml
@@ -284,7 +284,6 @@
             <include name="**/MessageTransformer.java"/>
           </fileset>
 	  <sourcepath>
-	    <path location="${ws.top.dir}/main/javax-jms/src/main/java"/>
             <path location="${ws.top.dir}/main/http-tunnel/tunnel-api-share/src/main/java"/>
 	    <path location="${ws.top.dir}/main/http-tunnel/tunnel-api-server/src/main/java"/>
 	    <path location="${ws.top.dir}/main/http-tunnel/tunnel/src/main/java"/> 
@@ -295,7 +294,6 @@
 	    <path location="${ws.top.dir}/main/persist/txnlog/src/main/java"/> 
  	    <path location="${ws.top.dir}/main/portunif/src/main/java"/> 
 	    <path location="${ws.top.dir}/main/jaxm-api/src/main/java"/> 
-	    <path location="${ws.top.dir}/main/javax-jms/src/main/java"/> 
 	    <path location="${ws.top.dir}/main/bridge/bridge-api/src/main/java"/> 
 	    <path location="${ws.top.dir}/main/bridge/bridge-admin/src/main/java"/> 
 	    <path location="${ws.top.dir}/main/bridge/bridge-api/src/main/java"/> 

